app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java

```diff
@@ -29,4 +34,15 @@ public void showDialogFragment(ListenerDialogFragment listenerDialogFragment) {
     public void showNumberOfParking(String numberOfParking) {
         binding.textViewMainActivity.setText(numberOfParking);
     }
+
+    @Override
+    public void showReservationActivity() {
+        Intent i = new Intent(getContext(), ReservationActivity.class);
+        Objects.requireNonNull(getActivity()).startActivity(i);
```

Comment:
why you use this function? `requireNonNull()` What happens if something inside returns null?

Reply:
done